### ISSUE TITLE
Is there any way to implement this plugin to the ShopAPI?

### ISSUE THREAD

**Issue by tudorradubarbu** [mollie/sylius#3]
I've been trying to implement the `CaptureAction` in an endpoint in the ShopAPI but no chance. What is a best-practice to add the payment redirection when reaching `complete_checkout` endpoint? 

Actually, is this possible (with the Mollie plugin)?

**Comment by nithiz** [comment]
@tudorradubarbu  Did you ever get this to work?

**Comment by tudorradubarbu** [comment]
Not really. BUT. Based on this plugin, I've used the `$this->mollieApiClient ` and used it in my actions/handlers. Super easy to understand if you check the code.

**Comment by nithiz** [comment]
@tudorradubarbu  Great thanks!
